test_unified_integration.py

```
@@ -0,0 +1,59 @@
+        # Test pipeline creation
+        pipeline = UnifiedIngestionPipeline(config)
+        print("✅ Pipeline created successfully")
+        
+        # Test initialization (but don't actually connect to avoid dependency issues)
+        print("✅ Pipeline initialization interface available")
+        
+        # Test source type detection
+        test_cases = [
+            ("https://github.com/user/repo", ContentType.GITHUB_REPOSITORY),
+            ("https://example.com", ContentType.WEB_PAGE),
+            ("/tmp/test", ContentType.LOCAL_FOLDER)
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)